tests/test_systems/test_spectrometers/test_HyperspectralImager.py

```diff
@@ -79,17 +85,22 @@ def test_get_optical_spatial_resolution():
 
 
 def test_get_sensor_spatial_resolution():
-    """Test get_sensor_spatial_resolution."""
-
+    """Test the sensor-limited spatial resolution method."""
```

Comment:
If we rename the docstring to match the function name, we can ensure the docstring is updated if the function name itself is ever changed
```suggestion
    """Test get_sensor_spatial_resolution method."""
```